Branch-SDK/Branch-SDK/BNCSystemObserver.m

```diff
@@ -121,13 +121,16 @@ + (NSString *)getDeviceName {
 }
 
 + (NSNumber *)getUpdateState {
-    NSString *bundleRoot = [[NSBundle mainBundle] bundlePath];    NSFileManager *manager = [NSFileManager defaultManager];
```

Comment:
If this returns an `NSNumber` if you need an update, and `nil` if you don't, wouldn't it just be better to make this a `BOOL` function, with a more appropriate name like "needsUpdate" or something?